docs/pages/enterprise/introduction.mdx

```diff
@@ -4,11 +4,36 @@ description: Introduction to features and benefits of using Teleport Enterprise.
 h1: Teleport Enterprise
 ---
 
-Teleport Enterprise is a commercial product built around Teleport's open source
-core.
+Teleport Enterprise is a self-hosted deployment of Teleport that allows
+organizations to achieve the most stringent standards of security and
+compliance. We offer Teleport Enterprise as a commercial product built around
+Teleport's open source core.
```

Comment:
```suggestion
Teleport's open source core.  Teleport Cloud is a deployment of the Teleport Enterprise Auth Service and Proxy Service, managed by a dedicated team at Teleport.  
```
I would have some quick definition of what Teleport Cloud is.  Essentially it's the enterprise software, deployed and managed by a Teleport team.

Reply:
@ptgott ^

Reply:
I'm not sure the introductory paragraph of this page is the best place for that sentence, since this page is about . I think the self-hosted aspect is a key differentiator for Teleport Enterprise, e.g., since you can configure it to use an HSM, use a FIPS-enabled binary, etc. If we equate Teleport Enterprise with Teleport Cloud, we lose those distinctions.